supplementary_materials/qwen2/requirements.txt

```diff
@@ -0,0 +1,7 @@
+nncf==2.10.0
+optimum-intel>=1.16.0
+transformers>=4.38.2
+openvino>=2024.1.0
```

Comment:
can we use nightly for obtaining the latest optimizations?

Reply:
no, as my test, the latest nightly build has accuracy issue on qwen2